packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1470,6 +1471,13 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.widgets.SelectionOverlay.onSelectionHandleTapped}
   final VoidCallback? onSelectionHandleTapped;
 
+  /// {@template flutter.widgets.editableText.groupId}
+  /// Assign a separate click area to each text field, groupId object.
+  ///
+  /// If this property is null, [EditableText] will be used.
+  /// {@endtemplate}
+  final Object? groupId;
```

Comment:
I wonder, if the default is `EditableText` anyway, can we make this non-nullable and use `EditableText` in the constructors of `EditableText`, `TextField`, `TextFormField`, as default value ?

Reply:
The constructor of TextFormField can accept default values, but making the groupId of EditableText mandatory will require callers to pass parameters, potentially necessitating modifications to other unit tests.

Reply:
I think I should understand what you mean